src/Playwright/Scripts/playwright.cmd

```diff
@@ -1,6 +1,6 @@
 @echo off
 setlocal
 if not defined PLAYWRIGHT_NODEJS_PATH (
-  set PLAYWRIGHT_NODEJS_PATH=%~dp0\node.exe
+  set PLAYWRIGHT_NODEJS_PATH="%~dp0\node.exe"
 )
-"""%PLAYWRIGHT_NODEJS_PATH%""" "%~dp0\..\..\package\lib\cli\cli.js" %*
```

Comment:
Does it make any difference compared to the previous script?

Reply:
https://github.com/microsoft/playwright/pull/19801